pyproject.toml

```diff
@@ -109,6 +109,7 @@ dependencies = [
     "duckduckgo-search>=6.3.0",
     "langchain-elasticsearch>=0.2.0",
     "opensearch-py>=2.7.1",
+    "langchain-experimental>=0.3.2"
```

Comment:
We shouldn't need to add this as we have it as dependency in Langflow-base

Reply:
I see, you are right. I just removed it and kept it in the base. Thanks for the insight.